src/v2/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -85,7 +85,7 @@ const ViewingRoomHeaderLarge: React.FC<ViewingRoomHeaderProps> = props => {
           {title}
         </Sans>
 
-        <Metadata {...props} />
+        <RoomInfo {...props} />
```

Comment:
I found `Metadata` confusing here. Because on the App lever we refer to `ViewingRoomMeta` and that is something different. Here it is just some additional room info that we need to group into one row. Maybe `PartnerTimeInfo` would be even more descriptive:)